rules/windows/builtin/win_remote_service.yml

```diff
@@ -0,0 +1,44 @@
+action: global
+title: Remote Service Creation
+id: 4a3a2b96-d7fc-4cb9-80e4-4a545fe95f46
+status: experimental
+description: Detects remote execution via service creation on the destination host
+author: Jai Minton, oscd.community
+date: 2020/10/05
+references:
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+tags:
+    - attack.lateral_movement
+    - attack.persistence
+    - attack.execution
+    - attack.t1543.003 
+logsource:
+    product: windows
+    service: security, system
+detection:
+    selection1:
+        EventID: 4624
+        Logon_Type: 3
+    filter1: 
+        Source_Network_Address:
+            - '::1'
+            - '127.0.0.1'
```

Comment:
I think we have the same issue here as in https://github.com/Neo23x0/sigma/pull/1033#discussion_r501979040

4624 with logon type 3 is a normal event, and one of the noisiest ones in AD environments. 
I think it makes sense to again ask the author about the way to translate it to Sigma logic, or even about the possibility to do that. @Heirhabarov could you please help us figure out how we can proceed with Sigma rule development? 

<img width="1347" alt="Screenshot 2020-10-08 at 22 30 49" src="https://user-images.githubusercontent.com/13090109/95510256-ee7f9000-09b5-11eb-98a2-37e1aeb73e21.png">

Reply:
Definitely agree, will need to think a bit more about how this can be done (if at all) in Sigma. Can only think of a subsearch similar to [#1033 (comment)](https://github.com/Neo23x0/sigma/pull/1033#discussion_r502216566) at the moment, and even then I'm not sure how it would look in Sigma. Hoping one of the bright minds in this community would have an idea on how this could be improved.